vaultlocker/shell.py

```diff
@@ -220,6 +225,12 @@ def main():
         type=int,
         help="Time in seconds to continue retrying to connect to Vault"
     )
+    parser.add_argument(
+        '--config',
+        default=DEFAULT_CONF_FILE,
+        type=str,
+        help="Path to an alternative configuration file"
```

Comment:
I think 'Path to vaultlocker configuration file' is fine here - a default is provided which is sane and this just allows a user to switch to somewhere else.

Reply:
Help string modified :)